### ISSUE TITLE
having issue to fork

### ISSUE THREAD

**Issue by jash0013** [jash0013/mtm6201-week1#3]
hey wendy!
hope you are fine
actually when i fork the assignment form github page and i also copy the code and paste in visual studio code clone repository but my issue is that when i open my github login page that is jash0013/grid-layout with flex components and when i go to setting and click on pages am unable to get my go live link in github page @wendywarren
![1](https://github.com/jash0013/mtm6201-week1/assets/156237728/5753bbc7-ce34-4238-8c8f-a26ea9176e30)
![2](https://github.com/jash0013/mtm6201-week1/assets/156237728/d88f8a3b-fc41-4abc-9208-cfa5c7bf8a0f)
. please see my issue here am attaching screenshot for easy to see

**Comment by wendywarren** [comment]
Hi @jash0013,

Did you change the branch from **none** to **master**? If you haven't, try that and select save.

Let me know how that goes.

Thanks,
Wendy

**Comment by jash0013** [comment]
I haven't tried yet. Can you please tell me how to do that one

**Comment by wendywarren** [comment]
If you look up at your last screenshot (on the settings page) you see in the Pages section that there is a heading called Branch? The dropdown that is currently set to none should be changed to master. Once you change it and save, you will see that your site is deployed.

**Comment by jash0013** [comment]
![jj](https://github.com/jash0013/mtm6201-week1/assets/156237728/c70eda1f-066f-4384-a2a8-f280c31eed77)
i had changed from none to master still am not getting my link when i go to setting --- pages as like in week 1 here am attaching a week 1 setting --- pages and it show link of my user id
![vv](https://github.com/jash0013/mtm6201-week1/assets/156237728/cb1422c0-c6c5-4bb9-b212-949f017c3b4f)
can you plz add a zoom meeting if possible

**Comment by wendywarren** [comment]
I see it there on your repo page.

https://jash0013.github.io/grid-layout-with-flex-components/

**Comment by jash0013** [comment]
You are right. Now I get my link thankyou so much Wendy . Have a. Good day
... After writing code in visual studio code and push the changes then for
the assignment...... The next step is to go to setting pages and send the
link right ???

**Comment by wendywarren** [comment]
Exactly! :)
